### DISCUSSION TITLE
Zed on Flatpak not showing Rust code errors until file saved

### DISCUSSION THREAD

**Discussion by g3blv** [zed-industries/zed#16612] [Feature Requests]
Is there some way of getting errors in Rust code shown in "real-time" rather than on save?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
